xmake/modules/detect/sdks/find_matlab.lua

```diff
@@ -0,0 +1,51 @@
+
+function _find_matlab()
+    local _ret = {}
+    local matlabkey = "HKEY_LOCAL_MACHINE\\SOFTWARE\\MathWorks\\MATLAB"
+    local valuekeys = winos.registry_keys(matlabkey)
+    if #valuekeys == 0 then
+        return {}
+    end
+    local itemkey = valuekeys[1] .. ";MATLABROOT"
+    local itemvalue
+    try{
+        function ()
+            itemvalue = winos.registry_query(itemkey)
+        end,
+        catch{
+            function (errors)
+                return {}
+            end
+        }
+    }
+
+    _ret.Matlab_FOUND = true
+    _ret.Matlab_ROOT_DIR = itemvalue
+    _ret.Matlab_INCLUDE_DIRS = path.join(itemvalue,"extern","include")
```

Comment:
and here

Reply:
please rename Matlab_INCLUDE_DIRS to `includedirs`, we use `includedirs`, `linkdirs`, `links`, ...